src/clockkit.cs

```diff
@@ -105,27 +146,54 @@ interface CLKComplicationServer {
 
 		[Export ("extendTimelineForComplication:")]
 		void ExtendTimeline (CLKComplication complication);
+
+		[Watch (7, 0), NoiOS]
+		[Export ("reloadComplicationDescriptors")]
+		void ReloadComplicationDescriptors ();
 	}
 
 	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
 	interface CLKComplicationTemplate : NSCopying {
 
 		[NullAllowed, Export ("tintColor", ArgumentSemantic.Copy)]
 		UIColor TintColor { get; set; }
+
+		[Deprecated (PlatformName.iOS, 14, 0, message: "The default constructor is deprecated. Use the provided factories instead.")]
```

Comment:
"Use the provided factories instead." would be enough, the first part is already implied